documentation/release_notes.rst

```diff
@@ -367,3 +383,4 @@ Known Issues and Limitations
 .. _`restrictions and known limitations`: https://oneapi-src.github.io/oneDPL/overview.html#restrictions
 .. _`Tested Standard C++ API`: https://oneapi-src.github.io/oneDPL/api_for_dpcpp_kernels/tested_standard_cpp_api.html#tested-standard-c-api-reference
 .. _`Macros`: https://oneapi-src.github.io/oneDPL/macros.html
+.. _`2022.0 Changes`: https://oneapi-src.github.io/oneDPL/
```

Comment:
Is this the intended link, or is this supposed to point to some subsection?

Reply:
It is supposed to point to a new section which is currently on review in https://github.com/oneapi-src/oneDPL/pull/672